odl_video/urls.py

```diff
@@ -24,3 +24,7 @@
     url(r'^', include('cloudsync.urls')),
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
 ]
+
+handler403 = 'ui.views.permission_denied_view'
+handler404 = 'ui.views.page_not_found_view'
+handler500 = 'ui.views.error_view'
```

Comment:
nit: Please rename the views to contain the error code

Reply:
sure